src/goto-instrument/accelerate/acceleration_utils.h

```diff
@@ -30,6 +31,7 @@ Author: Matt Lewis
 #include "cone_of_influence.h"
 
 typedef std::unordered_map<exprt, exprt, irep_hash> expr_mapt;
+typedef std::list<exprt> expr_listt;
```

Comment:
This is used only once in this file so does not really need a `typedef`. Plus I think `std::list<exprt>` carries more information than `expr_listt`.

Reply:
Or alternatively, can the definition in `utils/expr.h` be used instead? It seems like `std::list<exprt>` is used all over the place in the cbmc repo, but the `expr_listt` typedef is already defined in that header file.

Reply:
I am wondering whether the typedef `expr_listt` provides much benefit? It's 6 characters shorter. Value would arise if anyone pondered swapping out `std::list` for some other implementation -- is that happening?

Reply:
For the specific instance here, however, I'd argue to keep the typedef: It's used 23 types in the accelerator code.